sunkit_image/tests/test_radial.py

```diff
@@ -194,6 +197,75 @@ def test_fig_fnrgf(smap):
     out.plot()
 
 
+@figure_test
+@pytest.mark.remote_data()
+def test_fig_rhef(smap):
+    radial_bin_edges = utils.equally_spaced_bins(0, 2, smap.data.shape[1])
+    radial_bin_edges *= u.R_sun
+
+    out = rad.rhef(smap, radial_bin_edges=radial_bin_edges, upsilon=0.35, method="scipy")
+
+    out.plot()
+
+
+@figure_test
+@pytest.mark.remote_data()
+def test_multifig_rhef(smap):
+    radial_bin_edges = utils.equally_spaced_bins(0, 2, smap.data.shape[1])
+    radial_bin_edges *= u.R_sun
+
+    # Define the list of upsilon pairs
+    upsilon_list = [
+        0.35,
+        None,
+        (0.1, 0.1),
+        (0.5, 0.5),
+        (0.8, 0.8),
+    ]
+
+    # Call the plotting functions
+    # Adjust the map data to avoid log of zero
+    sdata = smap.data
+
+    # Small constant to avoid log of zero
+    epsilon = 1e-2
+
+    # Adjust the data to avoid log of zero
+    data0 = np.log10(np.maximum(sdata - np.nanmin(sdata), epsilon)) ** 2
+
+    # Extract the coordinate ranges from the meta information
+    x_coords = smap.meta["cdelt1"] * (smap.data.shape[1] // 2)
+    y_coords = smap.meta["cdelt2"] * (smap.data.shape[0] // 2)
+    extent = [-x_coords, x_coords, -y_coords, y_coords]
+
+    # Create a figure with subplots for each upsilon pair plus the original map
+    fig, axs = plt.subplots(2, 3, figsize=(15, 10), sharey="all", sharex="all")
+    axs = axs.flatten()
+
+    # Plot the original map
+    axs[0].imshow(data0, origin="lower", extent=extent, cmap=mpl.colormaps["sdoaia171"])
+    axs[0].set_title("Log10(data)^2")
+
+    # Loop through the upsilon_list and plot each filtered map
+    for i, upsilon in enumerate(upsilon_list):
+        out_map = rad.rhef(smap, radial_bin_edges=radial_bin_edges, upsilon=upsilon, method="scipy")
+        data = out_map.data
+        axs[i + 1].imshow(data, origin="lower", extent=extent, cmap=mpl.colormaps["sdoaia171"])
+        axs[i + 1].set_title(f"Upsilon = {upsilon}")
+
+    # Adjust layout
+    plt.tight_layout()
+    return fig
+
+
+# Tests needed for RHEF:
+# > With and without bins given
+# > Various inputs for Upsilon
+# > plotting smaps natively
+# > Doing upsilon on a map that doesn't have RHE on it
+# > Make a notebook as an example of usage
```

Comment:
Are these still to do? We don't have to do all of these, we can turn this into an issue to track for the future.